src/frontend/pages/feilsider/ServerFeil.tsx

```diff
@@ -0,0 +1,62 @@
+import { BodyShort, Box, Button, HGrid, Heading, List, Page, VStack } from '@navikt/ds-react';
+import * as React from 'react';
+
+interface Props {
+    statusKode?: number;
+    eksternFagsakId?: string;
+    behandlingId?: string;
+}
+
+export const ServerFeil: React.FC<Props> = ({ statusKode, eksternFagsakId, behandlingId }) => {
+    return (
+        <Page.Block as="main" width="xl" gutters>
+            <Box paddingBlock="20 8">
+                <HGrid columns="minmax(auto,600px)" data-aksel-template="500-v2">
+                    <VStack gap="16">
+                        <VStack gap="12" align="start">
+                            <div>
+                                {!!statusKode && (
+                                    <BodyShort textColor="subtle" size="small">
+                                        Statuskode {statusKode}
```

Comment:
Denne ble ikke vist da jeg testet i pre-prod
Hva om vi endrer fra statuskode til httpStatus med eksempel = `500 Internal Server Error`
```suggestion
                                        {httpStatus} 
```